.ci/docker-compose-file/.env

```diff
@@ -7,6 +7,7 @@ INFLUXDB_TAG=2.5.0
 TDENGINE_TAG=3.0.2.4
 DYNAMO_TAG=1.21.0
 CASSANDRA_TAG=3.11.6
+OPENTS_TAG=9aa7f88
```

Comment:
Is there any comments about this magic image hash?

Reply:
This isn't a magic number just it last version tag